src/control/security/pem.go

```diff
@@ -0,0 +1,137 @@
+package security
+
+import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+const (
+	safePermissions os.FileMode = 0700
+)
+
+func loadPEMData(filePath string) ([]byte, error) {
+	statInfo, err := os.Stat(filePath)
+	if err == nil {
```

Comment:
why not process err != nil case first and just continue with the other logic (outside the if/else clause) if not . that would be more in line with how we normally handle errors

Reply:
Good idea.